src/NuGet.Clients/PackageManagement.UI/PackageLoader.cs

```diff
@@ -485,7 +487,7 @@ private IEnumerable<string> GetSupportedFrameworks()
 
                 // if the package does not exist in the remote source, NuGet should
                 // try getting metadata from the local resource.
-                if (String.IsNullOrEmpty(metadata.Summary) && localResource != null)
+                if (String.IsNullOrEmpty(metadata?.Summary) && localResource != null)
```

Comment:
I think the idea is that online had more data like owner and download count. The change to enable it will be more massive, it will mean first showing the list of packages with the available local metadata, then getting the remote data and overlaying it lazily.